### PR TITLE
[4.0] Replace fixed width with class

### PR DESCRIPTION
**Pull request by Quy**
### Summary of Changes
Make 1st column width of 2nd and 3rd tabs consistent with 1st tab.

### Testing Instructions
Go to System > System Information.
Click on the 2nd, 3rd, and 4th tabs.

### CODE REVIEW

File: administrator/components/com_admin/tmpl/sysinfo/default_directory.php [hunk 1]
```diff
@@ -22,7 +22,7 @@
 		</caption>
 		<thead>
 			<tr>
-				<th scope="col" style="width:850px">
+				<th scope="col" class="w-25">
```
Comment (SharkyKZ) [inline administrator/components/com_admin/tmpl/sysinfo/default_directory.php]: This should be wider.

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on ea02f3d329fb2537dc3e69b84f2f20254abb877f<br /><br />Works fine.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29103">issues.joomla.org/tracker/joomla-cms/29103</a>.</sub>

File: administrator/components/com_admin/tmpl/sysinfo/default_directory.php (hunk 1 above)
Comment (Quy) [reply administrator/components/com_admin/tmpl/sysinfo/default_directory.php]: Would it be better to swap the columns?

Comment (SharkyKZ) [reply administrator/components/com_admin/tmpl/sysinfo/default_directory.php]: No, the column order is fine.

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on ea02f3d329fb2537dc3e69b84f2f20254abb877f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29103">issues.joomla.org/tracker/joomla-cms/29103</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 5c79a63c5369d05aeff9345159b19176deb4beb1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29103">issues.joomla.org/tracker/joomla-cms/29103</a>.</sub>

**Comment by richard67** [comment]
@Quy The first column on the 3rd tab still has a bigger width than it has on the the first two tabs in case if memcached is available, because of the long names of the memcached options. But that's ok for me. Just report back here if you wanna change that or not. If so, I will wait for that, and if not, I can give this PR a good test.

**Comment by Quy** [comment]
Will `w-30` 30% be fine for the 3 tabs?

**Comment by richard67** [comment]
@Quy Yes, w-30 is ok in my case. No idea of there are other normally hidden options which need more space. Other question: Any idea why drone is always failing at the composer step?

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on f5ddd1ea70a7cf1b5089cafce23ec5f1a47817f8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29103">issues.joomla.org/tracker/joomla-cms/29103</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on f5ddd1ea70a7cf1b5089cafce23ec5f1a47817f8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29103">issues.joomla.org/tracker/joomla-cms/29103</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29103">issues.joomla.org/tracker/joomla-cms/29103</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**